cellxgene_schema_cli/cellxgene_schema/remove_labels.py

```diff
@@ -30,23 +30,33 @@ def remove_labels(self):
             if component is None:
                 continue
 
+            component_def = self.schema_def["components"][component_name]
+
             # Doing it for columns
-            if "columns" in self.schema_def["components"][component_name]:
-                for column_def in self.schema_def["components"][component_name]["columns"].values():
+            if "columns" in component_def:
+                for column_def in component_def["columns"].values():
                     if "add_labels" in column_def:
                         self._remove_columns(component, column_def)
 
             # Remove automatically annotated columns
-            if "reserved_columns" in self.schema_def["components"][component_name]:
-                for field in self.schema_def["components"][component_name]["reserved_columns"]:
+            if "reserved_columns" in component_def:
+                for field in component_def["reserved_columns"]:
                     del component[field]
 
             # Doing it for index
-            if "index" in self.schema_def["components"][component_name]:
-                index_def = self.schema_def["components"][component_name]["index"]
+            if "index" in component_def:
+                index_def = component_def["index"]
                 if "add_labels" in index_def:
                     self._remove_columns(component, index_def)
 
+            # Remove any labels added as dict keys
```

Comment:
this is the bulk of the logic change to `remove_labels`, to just strip off any labels that were added as part of `uns`